docs/section10/10-7.md

```diff
@@ -0,0 +1,71 @@
+**Mandatory Rule:** True     
+**Rule ID:** 10-7  
+ 
+**Rule Description:** Baseline shall be modeled with the COPnfcooling HVAC system efficiency per Tables G3.5.1-G3.5.6.   
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1.2.1 Equipment Efficiencies  
+**Appendix G Section Reference:** None  
+**Data Lookup:** Tables G3.5.1 and G3.5.2  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem data table  
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 1, 1b, 2, 3, 3b, 4, 5, 5b, 6, and 6b that do not serve more than one floor.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_hvac_systems_5_6_serving_multiple_floors_b()
+
+**Manual Check:** None  
+
+## Rule Logic:  
+**Applicability Check 1:**   
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get list of hvac systems of type 5 and 6 that serve more than one floor: `hvac_sys_serve_more_than_one_flr_list_b = get_hvac_systems_5_6_serving_multiple_floors_b(B_RMR).keys()`
+    - For each hvac system in the the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
+        - Reset boolean variable that indicates if the hvac system is type 3 or 4 (used in rule assertion): `sys_type_3_or_4 = FALSE`
+        - Reset boolean variable that indicates if the hvac system is in the smallest capacity bracket (used in rule assertion): `smallcapBracket = FALSE`
+        - Get the hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+        - Check if HVAC system is of type 1, 1b, 2, 3, 3b, 4, 5, 5b, 6, or 6b AND that is does not serve more than one floor, if meets criteria then carry on: `if hvac_sys_type_b in ["SYS-1", "SYS-1b", "SYS-2", "SYS-3", "SYS-3b", "SYS-4","SYS-5", "SYS-5b", "SYS-6", "SYS-6b"] and hvac_b.id not in hvac_sys_serve_more_than_one_flr_list_b:` 
+            Get the attributes of the cooling system associated with this HVAC (e.g. efficiency, capacity, etc.)
+            - Get number of cooling systems, there should only be one cooling system - if there is more than one then the outcome of this RDS is UNDETERMINED: `num_cooling_sys = len(hvac_b.cooling_system)` 
+            - Assign the cooling system object to a variable: `cooling_system_b = hvac_b.cooling_system[0]`
+            - Get the total_cool_capacity: `total_cool_capacity_b = cooling_system_b.total_cool_capacity * CONVERSTION (W_TO_BTUH)`
+            - Get the full load cooling efficiency: `full_load_efficiency_b = cooling_system_b.full_load_efficiency`
+            Based upon the system type and cooling capacity determine the efficiency that is expected to be modeled
+            - Check if the system is system type 1 or 1b: `if hvac_sys_type_b = "SYS-1" or hvac_sys_type_b = "SYS-1b":`
```

Comment:
Going forward, instead of explicitly setting capacity brackets, I recommend referencing the appropriate table in App G for each baseline system type.

Reply:
Ok.